torch_xla/__init__.py

```diff
@@ -147,9 +147,14 @@ def _setup_tpu_vm_library_path() -> bool:
 
 
 def _prepare_to_exit():
-  _XLAC._prepare_to_exit()
-  if int(os.environ.get('PT_XLA_DEBUG', '0')):
-    _summarize_fn_tracker()
+  try:
+    _XLAC._prepare_to_exit()
+    if int(os.environ.get('PT_XLA_DEBUG', '0')):
+      _summarize_fn_tracker()
+  except Exception as e:
+    logging.error(
+        "Caught an exception when exiting the process. Exception: ", exc_info=e)
+    os._exit(1)
```

Comment:
sorry, one more micro-nit: make this `exit(1)`

Reply:
I tried and it didn't set the exit code correctly. Per https://docs.python.org/3/library/constants.html#exit, calling `exit(1)` raises a `SystemExit` exception. But due to https://bugs.python.org/issue27035, raising an exception won't set the exit code correctly.

Also, per https://docs.python.org/3/library/constants.html#exit, `exit` or `quit` "are useful for the interactive interpreter shell and should not be used in programs.".

I'll add a comment.